### PR TITLE
1) Refactor: Authentication

### LINKED ISSUE #1: 1) Refactor: Authentication
We’re currently storing tokens in localStorage, and would like to move to storing them in httpOnly cookies to improve security.

### PR DESCRIPTION
**Pull request by mushcore**
resolves #1 
change tokens to be stored in httpOnly cookies

### CODE REVIEW

File: server/routes/auth/index.js
Comment (Tahoum) [inline server/routes/auth/index.js]: You need to set an expiration date that matches the `JWT`

File: server/routes/auth/index.js
Comment (Tahoum) [inline server/routes/auth/index.js]: You can remove the token from the response since you have it set in the cookie.

**Review by Tahoum — COMMENTED** [review]
Good work on this one 💯 
Please address the comments and when you're done let us know in the channel so we can give it a second review 🙂

File: server/routes/auth/index.js
Comment (Mayo19) [inline server/routes/auth/index.js]: The [documentation for `res.clearCookie`](http://expressjs.com/en/api.html#res.clearCookie) says that compliant browsers/clients will only clear the cookie if the provided `options` are identical to those given when setting the cookie with `res.cookie()` (aside from `expires` and `maxAge`).

**Review by Mayo19 — APPROVED** [review]
Looks good, just left a one comment. I'm approving this so once you've addressed it feel free to merge 🙂

**Pull request merged**
